packages/react-ui/src/components/stack/index.js

```diff
@@ -3,6 +3,23 @@ import PropTypes from 'prop-types'
 import { Element } from '../../primitives'
 import { merge } from '../../utils'
 
+// TODO: prefixed version can be removed after upgrading to Emotion 11
+const createGap = (direction, gap) => {
+  if (direction === 'vertical') {
+    return {
+      marginBottom: gap,
+      '-webkitMarginEnd': 0,
```

Comment:
curious what browsers are we targeting with the -webkit prefix? According to the [caniuse](https://caniuse.com/#feat=css-logical-props) report you shared, it's needed for a fairly old version of chrome

Reply:
I'm more concerned about iOS Safari than Chrome - non-prefixed version is "just" supported since 2019. The whole thing is subjective though - it really depends on which browser **you** want to support. Happy to adjust this however you want.

Reply:
Ah! That's an interesting nuance! It's hard to say without defining a list of _supported browsers_ up front. 

I'd say RTL + older version of webkit is a combination we can ignore. Meanwhile, I can work on setting the boundary of what isn't supported.

Reply:
> I'd say RTL + older version of webkit is a combination we can ignore.

Can we though? If you switch to logical properties then it's not really affecting just RTL+older webkit combination but rather older webkit as is, because the margin won't be applied even for non-RTL layouts.

Reply:
Oh right, my bad!